native/build.go

```diff
@@ -115,7 +115,7 @@ func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 		return libcnb.BuildResult{}, fmt.Errorf("unable to find required manifest property\n%w", err)
 	}
 
-	command := filepath.Join(context.Application.Path, startClass)
+	command := "./" + startClass
```

Comment:
```suggestion
	command := fmt.Sprintf("%c%c%s", '.', os.PathSeparator, startClass)
```

It probably doesn't matter a ton, but I feel like we should probably use the path separator const. It's also a bit of a pain to work with cause it's a rune type not a string, so you can't just `"." + os.PathSeparator + startClass`. This should work though.